server/player/player.go

```diff
@@ -1888,7 +1888,10 @@ func (p *Player) PickBlock(pos cube.Pos) {
 	if pi, ok := b.(block.Pickable); ok {
 		pickedItem = pi.Pick()
 	} else if i, ok := b.(world.Item); ok {
-		it, _ := world.ItemByName(i.EncodeItem())
+		it, ok := world.ItemByName(i.EncodeItem())
```

Comment:
This is a workaround. `EncodeItem` should always be returning a valid registered item id & meta.

Reply:
I swear to you, it crashed my server. For some reason it doesn't

Reply:
That sounds like an issue to debug. Could you find out when that happens?